### PR TITLE
Don't pin to node v12

### PR DESCRIPTION
**Pull request by maresb**
The [code-server readme](https://github.com/cdr/code-server) says "at least node v12".

Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by maresb** [comment]
BTW, thanks so much @xhochy for the quick merge on the last PR. I really admire your many contributions!

**Comment by xhochy** [comment]
Tests passed using node 14, so this seems to work.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,14 +14,14 @@ source:
...
-  number: 0
+  number: 1
   binary_relocation: False  # [osx]
   skip: true  # [win]
 
 requirements:
   host:
   run:
-    - nodejs 12.*
+    - nodejs =>12
```
Comment (xhochy) [inline recipe/meta.yaml:24]: Can you use `>=` here? That's the more common form of it.

Comment (maresb) [reply recipe/meta.yaml:24]: I honestly don't know what I'm doing! I just saw it in https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#build-version-spec . (See "EXAMPLE: >=2,<3")

Comment (maresb) [reply recipe/meta.yaml:24]: Ah, I didn't see that typo until now!

Comment (maresb) [reply recipe/meta.yaml]: Ok, I think I fixed it now. You can see my new commit, right?

Comment (xhochy) [reply recipe/meta.yaml]: Yes, looks right

**Pull request merged**